include/crocoddyl/core/solvers/ipopt/ipopt.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef CROCODDYL_CORE_SOLVERS_IPOPT_IPOPT_HPP_
+#define CROCODDYL_CORE_SOLVERS_IPOPT_IPOPT_HPP_
+
+#include "coin-or/IpIpoptApplication.hpp"
+
+#include "crocoddyl/core/solver-base.hpp"
+#include "crocoddyl/core/solvers/ipopt/multiple-shooting-nlp.hpp"
+namespace crocoddyl {
+class SolverIpOpt : public SolverAbstract {
```

Comment:
As mentioned above, please rename this function to `SolverIpopt`.
Also add the line between the header and namespace (sorry for this picky comment ;))